config/packages/framework.yaml

```diff
@@ -1,5 +1,4 @@
 framework:
-    secret: '%env(APP_SECRET)%'
```

Comment:
I think the core-upgrade/install process does something with the secret? can't remember.

Reply:
It only takes it from old installations to migrate it into the `env` file. If the parameter does not exist it generates a new secret instead. So no problem to remove it for new installs.